Content.Server/Kitchen/Components/ActiveMicrowaveComponent.cs

```diff
@@ -11,6 +11,9 @@ public sealed partial class ActiveMicrowaveComponent : Component
     [ViewVariables(VVAccess.ReadWrite)]
     public float CookTimeRemaining;
 
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float ElapsedTime;
```

Comment:
Generally standard to instead store the TimeSpan at which you started counting, and calculate the elapsed using IGameTiming's curtime.

Reply:
Additionally make sure to use a timespan serializer, as to not break saving active microwaves. (which might be broken anyway who knows but best not make it worse)